README.md

```diff
@@ -26,6 +26,7 @@ macOS: https://github.com/woodruffw/ff2mpv/wiki/Installation-on-macOS
 
 Windows: https://github.com/woodruffw/ff2mpv/wiki/Installation-on-Windows
 
+[A Rust version of the native client is also available](https://github.com/ryze312/ff2mpv-rust).
 [A Go version of the native client is also available](https://git.clsr.net/util/ff2mpv-go/).
```

Comment:
Two third party clients makes me think we have enough for a list :slightly_smiling_face: 

```suggestion
The following community-maintained native clients are also available:

* [Rust](https://github.com/ryze312/ff2mpv-rust)
* [Go](https://git.clsr.net/util/ff2mpv-go/)
```